bench/workgen/workgen.cxx

```diff
@@ -553,7 +568,7 @@ void ThreadRunner::op_create_all(Operation *op, size_t &keysize,
     tint_t tint;
 
     op->create_all();
-    if (op->_optype != Operation::OP_NONE) {
+    if (op->is_table_op()) {
```

Comment:
Line #696 should have the same check. Throttle applies only to insert, read and update.